laravel/advanced-guides/global-sqlite-litefs.html.md.erb

```diff
@@ -0,0 +1,307 @@
+---
+title: Global SQLite with LiteFS
+layout: framework_docs
+objective: Go LiteFS-global with your SQLite-backed Laravel Fly App!
+---
+
+So you've decided to give SQLite a chance huh? If your answer's a bold, brave <b><i>yes</i></b>, then welcome! There's priceless <b>walkthough treasures</b> waiting for your read below! 
+
+On the other hand, if your answer is a resounding, mundane no, you might want to rethink your answer champ. You <i>are</i> here for a reason, so read on!
+
+
+## _Global SQLite--But How?_
+ 
+Admitably, SQLite's <i>a bit</i> embedded, and you might think there's nary a quick way in getting your Laravel-SQLite Fly App running multiple instances, oceans apart, running in different regions with all their wonderful data in-sync.
```

Comment:
Admittedly instead of Admitably ;)

Reply:
yooo @Johannes-Fly finally! cool, will fix this one, thanks!

Reply:
fixed!